confidential-identity/Cargo.toml

```diff
@@ -73,7 +73,7 @@ alloc = [
 	"curve25519-dalek/alloc",
 ]
 
-no_std = [ ]
+no_std = [ "u64_backend" ]
```

Comment:
Isn't `no_std` wasm? does wasm have a u64 backend?

Reply:
It is just a move from line `32`. I think it is more _explicit_ here than inside `curve25519-dalek`.
It works on "wasmi" impl that we use with substrate. 
I _guess_ this also works in `wasm-bingen`, because it does not modify the back-end, as I said, it only makes more explicit.